src/index.html

```diff
@@ -8,42 +8,45 @@
     <link rel="stylesheet" href="./styles.css">
   </head>
   <body>
-
+    <section class="user-info">
+      <h1>Hi, I'm John</h1>
+      </section>
     <header class='main-header'>
       <div class="wrapper">
         <img class="hamburger-menu" src="./images/hamburger-icon.png" alt="hamburger icon serving as menu button">
-        <img class="x-menu hidden" src="./images/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
```

Comment:
We should probably plan to have these `.hidden` classes on our elements by default, then remove them on a toggle when the hamburger icon is clicked.